Bento/StyleSheets/StackViewStyleSheet.swift

```diff
@@ -1,31 +1,43 @@
 import UIKit
 
-open class StackViewStyleSheet: ViewStyleSheet<UIStackView> {
+open class StackViewStyleSheet<View: UIStackView>: StyleSheetProtocol {
```

Comment:
Can we have a doc-comment for this class as mentioned in PR body?
(especially why we don't inherit `ViewStyleSheet` )

Reply:
@inamiy I've included documentation to explain it.